collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -522,16 +522,19 @@ private boolean uploadOneSubmission(String urlString, String id, String instance
     protected Outcome doInBackground(Long... values) {
         Outcome outcome = new Outcome();
 
-        String selection = InstanceColumns._ID + "=?";
+        StringBuffer selectionBuf = new StringBuffer(InstanceColumns._ID + " IN (");
         String[] selectionArgs = new String[(values == null) ? 0 : values.length];
-        if (values != null) {
+        if(values != null) {
```

Comment:
There should be a space between the if and the opening paren! I think that's a generally-accepted convention for C-like languages and [here](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace) it is from Google's conventions. We follow the Android open source style guide.